crates/loans/src/types.rs

```diff
@@ -1,8 +1,25 @@
-use crate::InterestRateModel;
+use crate::{Config, InterestRateModel};
+use currency::Amount;
 use frame_support::pallet_prelude::*;
-use primitives::{CurrencyId, Rate, Ratio};
+use primitives::{CurrencyId, Liquidity, Rate, Ratio, Shortfall};
 use scale_info::TypeInfo;
 
+/// Container for account liquidity information
+#[derive(Encode, Decode, Eq, PartialEq, Clone, RuntimeDebug, TypeInfo)]
```

Comment:
Is this exposed anywhere? If it is not used in storage/extrinsics, there is no need to derive `Encode`, `Decode` and `TypeInfo`. And if it's only used within the the pallet please make it `pub (crate)`.

Reply:
Good tips, the type to be public though